docs/drafts/support-watchNamespaces.md

```diff
@@ -0,0 +1,24 @@
+# Install Modes and WatchNamespaces in OMLv1
```

Comment:
My understanding is that this implements [this brief](https://docs.google.com/document/d/1EGcJUjEqyA6CVfRHP5EFzdAJZ8EZAv5oBU5wTJfjhso/edit). Is this correct? Did we have an RFC for this or just the brief?

Reply:
I believe only the brief. I think @varshaprasad96 was already working in this area before the brief was "accepted"

Reply:
It does so partially - in the sense we support specifying watchNamespaces only for registryV1. We only have brief for this at the moment. 

Additionally, there were changes which were done in https://github.com/operator-framework/rukpak/pull/774 that have been pulled in this PR - which involves:
- Removing bundle API
- Modifying the BD spec accordingly
- Introducing `watchNamespaces` to support all other install strategies - connected to the brief you mentioned

Reply:
> We only have brief for this at the moment.

> I believe only the brief. I think @varshaprasad96 was already working in this area before the brief was "accepted"

That is ok. The brief is very brief.

Thanks for the clarification both. Will resolve this thread.